SnapKit.xcodeproj/project.pbxproj

```diff
@@ -533,7 +533,7 @@
 				SKIP_INSTALL = YES;
 				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 4.2;
+				SWIFT_VERSION = 5.0;
```

Comment:
I don't know if we should force people to use Swift 5.0.
The project should resolve every warning it currently has against it, but the sample project should still run on the lowest version Xcode 10 supports in my opinion (e.g Swift 4.2).